supa.env

```diff
@@ -69,10 +69,10 @@
 #scheduler_max_workers=12
 
 # Name of the domain SuPA is responsible for.
-#domain=netherlight.net:2013
```

Comment:
is this just a default? (e.g. overridden on deploy?)

Reply:
The supa.env file is intended to be an example. The defaults for the parameters from this file are set in supa/__init__.py. When deployed a deployment specific environment file must be created. I did however changed all netherlight.net references to example.domain.